src/ansys/units/dimensions.py

```diff
@@ -26,7 +26,7 @@ def __init__(
         if units and dimensions:
             raise DimensionsError.EXCESSIVE_PARAMETERS()
 
-        self._units = ansunits.Units()
+        self._units = ansunits
```

Comment:
I'd just remove this member variable entirely and replace all the usage with direct calls to functions provided from the new module.